test/unit/component/parser/json/BpmnJsonParser.flowNode.test.ts

```diff
@@ -449,7 +449,7 @@ describe.each([
       });
     });
 
-    it(`should convert as Shape with incoming/outgoing attributes calculated from ${bpmnKind} attributes and from flows`, () => {
+    it(`should convert as Shape with incoming/outgoing attributes calculated from flows only`, () => {
```

Comment:
Rename into `should convert as Shape with incoming/outgoing attributes only calculated from flows`